process-engine/process_engine_core
# 🔥Remove error boundary event name matching

## Changes

The `ErrorBoundaryEventHandler` no longer matches the name of a given error against the error attached to the BoundaryEvent.
This means that an error now only has to have a matching code and a message.

Part of https://github.com/process-engine/process_engine_runtime/issues/497

PR: #316

## How to test the changes

- Use an `ErrorBoundaryEvent` that uses a random value for the `name` property and attach it to any kind of activity (easiest would be a ScriptTask)
- Cause an error that uses a different name from the error on the `ErrorBoundaryEvent`
- See that the `ErrorBoundaryEvent` can handle the error
